odl/tomo/backends/astra_cuda.py

```diff
@@ -228,37 +229,72 @@ def astra_cuda_back_projector(proj_data, geometry, reco_space, out=None):
 
     # Angular integration weighting factor
     # angle interval weight by approximate cell volume
-    extent = float(geometry.motion_partition.extent())
-    size = float(geometry.motion_partition.size)
-    scaling_factor = extent / size
+    angle_extent = float(geometry.motion_partition.extent())
+    num_angles = float(geometry.motion_partition.size)
+    scaling_factor = angle_extent / num_angles
```

Comment:
You mean the whole thing or just these 3 lines? The whole thing would make sense.